### ISSUE TITLE
[BUG] Protobuf gencode console warning messages

### ISSUE THREAD

**Issue by ZeLonewolf** [onthegomap/planetiler#1026]
There are console warning messages as follows:

```
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Decl$FunctionDecl$Overload. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Decl$FunctionDecl. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Type$ListType. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Expr. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Expr$Ident. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Expr$Select. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Expr$CreateList. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Expr$Call. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.SourceInfo. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.SyntaxProto. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.ParsedExpr. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Type$FunctionType. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.DeclProto. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.Reference. Please avoid checked-in Protobuf gencode that can be obsolete.
Sep 22, 2024 1:18:14 PM com.google.protobuf.RuntimeVersion validateProtobufGencodeVersionImpl
WARNING:  Protobuf gencode version 4.28.1 is older than the runtime version 4.28.2 at com.google.api.expr.v1alpha1.CheckedExpr. Please avoid checked-in Protobuf gencode that can be obsolete.
```

**Comment by msbarry** [comment]
Is this with planetiler bundled in an application with other dependencies including protobuf?  The planetiler build that gets distributed here has the right version of protobuf runtime packaged, but if you have a custom build that pulls in the wrong version it will cause this kind of issue.

**Comment by msbarry** [comment]
Oh actually it looks like this could be a dependency version issue between the version of protobuf planetiler directly depends on and the version CEL transitive depends on.

**Comment by ZeLonewolf** [comment]
Yes, this is running planetiler with a yaml config file containing CEL expressions.

**Comment by msbarry** [comment]
Fixed by #1030
